Logos/tenablead_logo.svg

```diff
@@ -0,0 +1,4 @@
+<svg id="5eb4e179-601a-4ee2-b232-fd4ded4f84cd" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 326 72.5">
```

Comment:
Logo size should be less than 5kb. Please update.

Reply:
Hi @v-sabiraj , I asked our graphic design team to make the logo less than 5kb, but it seems that's it's too much tiny..
Any possiblity that you make an exception and accept this logo please ?

Reply:
Hi @fr4nco1s , We can't accept this logo, it is mandatory to have logo size less than 5kb. Please update it.

Reply:
Hi @v-sabiraj , we managed to minify the svg logo size under 5kb.